configs/machines/levante.yaml

```
@@ -79,6 +79,7 @@ computer:
 
     module_actions:
             - "purge"
+            - "load python3"
```

Let's be specific about the version, because otherwise the users will install on a python version, and then wonder why esm_runscripts is not running anymore their tidying job after python load in the .run.
```suggestion
            - "load python3/2023.01-gcc-11.2.0"
```

This should be fixed once we sort out the python environments into a smarter way, but for now we need to be careful with the python versions